ccd-definition/CaseEventToComplexTypes/DrawDirectionsOrder-SDO-nonprod.json

```diff
@@ -0,0 +1,83 @@
+[
```

Comment:
Given my above comment, if you were to make everything related to the `CREATE_SDO` event then you would just need to change the filename here to `CreateSDO-SDO-nonprod.json` or something similar as well as the `CaseEventID` for all the fields below